tox.ini

```diff
@@ -1,11 +1,17 @@
 [tox]
+skip_missing_interpreters = True
 envlist =
   py{27,34,35,36}-django111-wagtail{112,113}
   py{34,35,36}-django{111,200}-wagtail{200,210}
 
+[pytest]
+norecursedirs = node_modules docs *.egg-info .git .tox
+
 [testenv]
 commands = pytest
 extras = test
+setenv =
+  DJANGO_SETTINGS_MODULE=wagtailautocomplete.tests.settings
```

Comment:
No tox changes in this PR, please! If you rebase on `master`, the changes in 1ec810d85c26bc6dcfb9b75aa550ce5378dd47a3 should make it easier to run tox locally. (I also don't want to quietly skip missing interpreters. It's totally reasonable to run it that way on your local machine, but this configuration file is used in CI, where we do want it to throw an error if it can't test all our specified environments.)